### PR TITLE
Fields messages

### PR DESCRIPTION
**Pull request by VISOSO2403**
# Description

- installed Formik and Yup for the validations of each entry
- added validations on each entry with its message in the worker and company form
- added two style variables in the global theme

# Images

Note:
When testing with the first form, the error messages came out (the team saw that everything worked fine).
I didn't take a picture and now it doesn't give the messages, even so, I upload the changes to see if in the emulators of the rest of the team it works fine.

### CODE REVIEW

File: src/components/auth/CompanyForm.tsx [hunk 1]
```diff
@@ -1,56 +1,132 @@
...
-    password,
-    setPassword,
-    setConfirmPass,
-    confirmPass,
-    handleEmailOrPhoneChange,
-  } = useAuth();
+const SigupCompanyScheme = Yup.object().shape({
+  name: Yup.string()
+    .min(4, 'Too short name')
+    .max(30, 'Too long name')
```
Comment (juperameza) [inline src/components/auth/CompanyForm.tsx]: Maybe try to change th writing on ths and the password, may someting like the name is too short
And how is a name too long yo're basically letting out a lot of peolple

Comment (juperameza) [reply src/components/auth/CompanyForm.tsx]: What if my company has more than 30 characters

File: src/components/auth/CompanyForm.tsx [hunk 2]
```diff
@@ -1,56 +1,132 @@
...
+    .required('Required'),
+  email: Yup.string().email('Invalid email').required('Required'),
+  password: Yup.string()
+    .min(8, 'Too short password')
+    .max(20, 'Too long password')
```
Comment (juperameza) [inline src/components/auth/CompanyForm.tsx]: Heres the same how is a password to long

File: src/components/auth/CompanyForm.tsx [hunk 3]
```diff
@@ -1,56 +1,132 @@
...
+            onPress={() => handleSubmit}
+            style={globalStyles.primaryBtn}>
+            <TextComponent
+              text="SIGN UP"
+              font="bold"
+              size={24}
+              color="white"
+              styles={{textAlign: 'center'}}
+            />
+          </TouchableOpacity> */}
```
Comment (juperameza) [inline src/components/auth/CompanyForm.tsx]: Dont leave commented code

File: src/components/auth/WorkerForm.tsx [hunk 4]
```diff
@@ -1,69 +1,162 @@
...
+  firstName: Yup.string()
+    .min(4, 'Too Short!')
+    .max(30, 'Too Long!')
+    .required('Required'),
+  lastName: Yup.string()
+    .min(4, 'Too Short!')
+    .max(30, 'Too Long!')
+    .required('Required'),
+  email: Yup.string().email('Invalid email').required('Required'),
+  // phone: Yup.number().min(10).max(10).required('Required'),
```
Comment (juperameza) [inline src/components/auth/WorkerForm.tsx]: dont elvea  commented code

File: src/components/auth/WorkerForm.tsx [hunk 5]
```diff
@@ -1,69 +1,162 @@
...
+  password: Yup.string()
+    .min(8, 'Too Short!')
+    .max(20, 'Too Long!')
```
Comment (juperameza) [inline src/components/auth/WorkerForm.tsx]: Same here hows a passwrd
too long

File: src/components/auth/WorkerForm.tsx [hunk 6]
```diff
@@ -1,69 +1,162 @@
...
+            />
 
-      <TextComponent styles={globalStyles.input} text="Password Confirmation" />
-      <InputComponent
-        value={confirmPass}
-        onChangeText={val => setConfirmPass(val)}
-        keyboardType="default"
-        secureTextEntry
-      />
+            {/* <DropdownField title="Gender" user={User} /> */}
```
Comment (juperameza) [inline src/components/auth/WorkerForm.tsx:123]: dont, leave commented code

**Review by juperameza — CHANGES_REQUESTED** [review]

File: src/components/auth/CompanyForm.tsx (hunk 1 above)
Comment (PudinDeveloper) [reply src/components/auth/CompanyForm.tsx]: @juperameza is right, for example even the legal name of BrightCoders has 37 characters: 

> "BRIGHTCODERS CONSULTING, S.A. DE C.V."

**Review by PudinDeveloper — APPROVED** [review]

**Review by Antonio3veces3 — APPROVED** [review]
Nice work!

File: src/components/auth/CompanyForm.tsx (hunk 1 above)
Comment (VISOSO2403) [reply src/components/auth/CompanyForm.tsx]: thank you for your comments.
I have made the changes

File: src/components/auth/CompanyForm.tsx (hunk 2 above)
Comment (VISOSO2403) [reply src/components/auth/CompanyForm.tsx]: thanks.
I have made the changes

File: src/components/auth/CompanyForm.tsx (hunk 3 above)
Comment (VISOSO2403) [reply src/components/auth/CompanyForm.tsx]: I have made the changes

File: src/components/auth/WorkerForm.tsx (hunk 4 above)
Comment (VISOSO2403) [reply src/components/auth/WorkerForm.tsx]: I have removed unnecessary comments

**Review by juperameza — APPROVED** [review]

**Pull request merged**